### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by salih101**

### CODE REVIEW

**Review by gbrozdev — APPROVED** [review]

**Comment by gbrozdev** [comment]
Njan review cheythukunu

**Review by gbrozdev — APPROVED** [review]

**Review by salih101 — COMMENTED** [review]
add h4

**Review by salih101 — COMMENTED** [review]
good

**Comment by salih101** [comment]
adsf

**Review by salih101 — COMMENTED** [review]
f

File: index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
           <li>www.google.com</li>
           <li>www.steam.com</li>
       <li>My favorite site is <a href="https://github.com">GitHub</a>!</li>
+          <h4>CHANGES</h4>
```
Comment (salih101) [inline index.html:13]: good

**Review by salih101 — COMMENTED** [review]
top

**Comment by gbrozdev** [comment]
Iyyentha ith kanikkunne 🤤🙂💔

**Review by gbrozdev — APPROVED** [review]

**Pull request merged**